var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -60,6 +63,8 @@ def fetch_remote_versions(self):
 
     def cmake_args(self):
         args = [
+            '-DBUILD_SHARED_LIBS:BOOL={0}'.format((
+                'ON' if '+shared' in self.spec else 'OFF')),
```

Comment:
`-DBUILD_SHARED_LIBS=%s' % variant_bool('+shared')`?

Reply:
Forget about that, it is paraview specific:
```
def variant_bool(feature, on='ON', off='OFF'):
            """Ternary for spec variant to ON/OFF string"""
            if feature in spec:
                return on
            return off
```
might be a useful function in general.